pkg/cmd/serve.go

```diff
@@ -47,6 +47,10 @@ func RegisterServeFlags(cmd *cobra.Command, config *server.Config) error {
 	config.DispatchClusterMetricsEnabled = true
 	config.DispatchClientMetricsEnabled = true
 
+	// Flags for logging
+	cmd.Flags().BoolVar(&config.EnableRequestLogs, "enable-request-logs", false, "logs API request payloads")
```

Comment:
Flags need to be named with a prefix for the subsystem they control, so that they are grouped together.
We should prefix this with `grpc` or `log`; probably log.

Any existing flags that start with `enable` were mistakes and should probably have their flags hidden and new ones introduced.

Reply:
`grpc-request-logs-enabled` `grpc-repsonse-logs-enabled`